src/Row/Row.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FC } from 'react'
+import styled, { css } from 'styled-components'
+
+import { theme } from '../theme'
+import { Icon } from '../Icon'
+import { Box } from '../Box'
+import { Text } from '../Text'
+
+export type Props = {
+  iconLeft?: string
+  iconRight?: string
+  iconRightColor?: string
```

Comment:
Should we add color also for the left one or do you think that will always be default?

Reply:
May as well!